sdk/core/azure-core-amqp/vcpkg/vcpkg.json

```diff
@@ -17,9 +17,7 @@
       "default-features": false,
       "version>=": "1.9.0-beta.1"
     },
-    {
-      "name": "azure-uamqp-c"
-    },
+    "azure-uamqp-c",
```

Comment:
I am surprised this is the format that is expected. That seems buggy. So, if we need to specify a version, or some other property, in the future, only then we get a `key:value` nested object for the dependency?

Reply:
If you have any other non-default property, you expand it into an object.
Otherwise, if it is name only, vcpkg-format expects it to be a single string.